### PR TITLE
Update Traditional Chinese for v15

### PR DESCRIPTION
**Pull request by dirtTW**
Update and Fix Traditional Chinese localization for v15
From https://github.com/shedaniel/cloth-config/pull/216 by @notlin4

### CODE REVIEW

File: common/src/main/resources/assets/cloth-config2/lang/zh_tw.json [hunk 1]
```diff
@@ -1,50 +1,52 @@
...
+    "text.cloth-config.dropdown.value.unknown": "§c無建議",
+    "text.cloth-config.disabled_tooltip": "已停用（未達到條件要求）",
+    "modifier.cloth-config.alt": "Alt + %s",
+    "modifier.cloth-config.ctrl": "Ctrl + %s",
+    "modifier.cloth-config.shift": "Shift + %s",
+    "title.cloth-config.config": "Cloth Mod Config 設定",
+    "category.cloth-config.scrolling": "捲動",
+    "category.cloth-config.testing": "預覽設定",
+    "option.cloth-config.scrollDuration": "滾動時間",
+    "option.cloth-config.scrollStep": "滾動距離",
```
Comment (notlin4) [inline common/src/main/resources/assets/cloth-config2/lang/zh_tw.json]:
```suggestion
    "option.cloth-config.scrollStep": "捲動距離",
```

File: common/src/main/resources/assets/cloth-config2/lang/zh_tw.json [hunk 2]
```diff
@@ -1,50 +1,52 @@
...
+    "option.cloth-config.bounceBackMultiplier": "反彈倍率",
+    "option.cloth-config.setDefaultSmoothScroll": "設為預設平滑滾動",
+    "option.cloth-config.disableSmoothScroll": "停用平滑捲動",
+    "text.cloth-config.testing.1": "你是模組開發者，想要使用 %1$s 嗎？前往 %2$s 開始吧。你也可以 %3$s。",
     "text.cloth-config.testing.2": "Cloth Config Wiki",
-    "text.cloth-config.testing.3": "點擊此處打開 Minecraft 選項文件，沒有特定原因"
-}
+    "text.cloth-config.testing.3": "點選此處開啟 Minecraft 選項檔案（無特殊原因）"
```
Comment (notlin4) [inline common/src/main/resources/assets/cloth-config2/lang/zh_tw.json]:
```suggestion
    "text.cloth-config.testing.3": "按這裡開啟 Minecraft 選項檔案（無特殊原因）"
```

File: common/src/main/resources/assets/cloth-config2/lang/zh_tw.json [hunk 3]
```diff
@@ -1,50 +1,52 @@
...
-  "option.cloth-config.bounceBackMultiplier": "反彈乘數",
-  "option.cloth-config.setDefaultSmoothScroll": "設置預設滑動",
-  "option.cloth-config.disableSmoothScroll": "禁用滑動",
-    "text.cloth-config.testing.1": "你是模組開發者並且想要使用 %1$s？前往 %2$s 以開始。你也可以 %3$s。",
+    "text.cloth-config.save_and_done": "儲存並退出",
+    "text.cloth-config.apply": "套用變更",
+    "text.cloth-config.quit_config": "變更尚未儲存",
+    "text.cloth-config.quit_config_sure": "確定要離開嗎？變更將不會被儲存！",
+    "text.cloth-config.cancel_discard": "放棄變更",
+    "text.cloth-config.quit_discard": "退出並放棄變更",
```
Comment (notlin4) [inline common/src/main/resources/assets/cloth-config2/lang/zh_tw.json]:
```suggestion
    "text.cloth-config.quit_discard": "離開並放棄變更",
```

**Review by notlin4 — CHANGES_REQUESTED** [review]

**Review by shedaniel — APPROVED** [review]

**Comment by shedaniel** [comment]
十分感謝！:D

**Pull request merged**